platforms/software/build-init/src/integTest/groovy/org/gradle/buildinit/projectspecs/internal/BuildInitPluginProjectSpecsIntegrationTest.groovy

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.buildinit.projectspecs.internal
+
+import org.gradle.api.JavaVersion
+import org.gradle.buildinit.plugins.AbstractInitIntegrationSpec
+import org.gradle.buildinit.plugins.TestsInitProjectSpecsViaPlugin
+import org.gradle.integtests.fixtures.AvailableJavaHomes
+import org.gradle.integtests.fixtures.jvm.JavaToolchainFixture
+import org.gradle.internal.jvm.Jvm
+import org.gradle.plugin.management.internal.autoapply.AutoAppliedPluginHandler
+import org.gradle.test.fixtures.file.LeaksFileHandles
+import org.gradle.test.precondition.Requires
+import org.gradle.test.preconditions.UnitTestPreconditions
+
+class BuildInitPluginProjectSpecsIntegrationTest extends AbstractInitIntegrationSpec implements TestsInitProjectSpecsViaPlugin, JavaToolchainFixture {
+    def "can specify 3rd party plugin using argument to init"() {
+        when:
+        initSucceedsWithPluginSupplyingSpec("org.barfuin.gradle.taskinfo:2.2.0")
```

Comment:
This plugin is completely arbitrary.  In the latest code, this is reflected in the name of the constant.  

What's the standard plugin called?  I tried looking in the mirrored repo under some obvious places and couldn't find it.  This can be whatever, just let me know what to use.